ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -344,23 +353,25 @@ def get_hindi_names_without_regex(self, text):
         """
         This method is used to get detect hindi names without any regex pattern (This method is called only if
         detection from regex patterns fails)
+        This method removes common hindi words ocurring in context of name and hindi stop words
+        COMMON_HINDI_WORDS_OCCURING_WITH_NAME set of hindi words ocurring in context of name
         Args:
             text (str): the text from which hindi text has to be detected
         Returns:
             person_name (tuple): two dimensional tuple
             1. entity_value (list): representing the entity values of names
             2. original_text (list): representing the original text detected
-
-
         Example:
             text = u'प्रतिक श्रीदत्त जयराओ'
             get_hindi_names_without_regex(text=text)
-
             >> [{first_name: u"प्रतिक", middle_name: u"श्रीदत्त", last_name: u"जयराओ"}], [ u'प्रतिक श्रीदत्त जयराओ']
-
         """
-
         text = self.replace_stopwords_hindi(text)
+        word_list = [word for word in text.split(" ") if word not in COMMON_HINDI_WORDS_OCCURING_WITH_NAME]
+        if word_list:
```

Comment:
text = " ".join([word for word in text.split(" ") if word not in COMMON_HINDI_WORDS_OCCURING_WITH_NAME])
if not text:
    return [], []